### PR TITLE
DX-1274: Features refactoring

### PR DESCRIPTION
**Pull request by marinafrydenb**

### CODE REVIEW

File: _includes/instrument-mode.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
+{% include api_rersource = include.api_resouce %}
+{% include documentation_section =include.documentation_section %}
```
Comment (asbjornu) [inline _includes/instrument-mode.md:2]: The correct syntax here seems to be `assign` and not `include`, but we've experienced that the best is to not `assign` the `include` parameters, but instead refer directly to i.e. `include.api_resouce` everywhere. So the best would be to simply remove these two lines and change the references from `api_resource` to `include.api_resource`.

```suggestion
```

**Review by asbjornu — CHANGES_REQUESTED** [review]

File: features/optional-features.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  Welcome to Optional Features.
+  This section has extented code examples and features that were not
+  covered by the other subsections.
+menu_order: 1000
+---
+
+{% include cobadge-dankort.md %}
+
+{% include instrument-mode.md api_resource=“paymentorders”
+documentation_section=“payment-menu” %}
```
Comment (asbjornu) [inline features/optional-features.md:15]:
```suggestion
{% include instrument-mode.md api_resource="paymentorders"
documentation_section="payment-menu" %}
```

**Review by asbjornu — CHANGES_REQUESTED** [review]

File: features/index.md [hunk 3]
```diff
@@ -0,0 +1,279 @@
+---
```
Comment (asbjornu) [inline features/index.md:1]: I'm not sure a `features/` folder at the root makes sense?

File: checkout/features/index.md [hunk 4]
```diff
@@ -0,0 +1,174 @@
+---
+title: Features
```
Comment (asbjornu) [inline checkout/features/index.md:2]: We need to add `redirect_from: /<documentation section>/other-features` in all `features/index.md` files so people with the old URL bookmarked is directed to the new location.

```suggestion
title: Features
redirect_from: /checkout/other-features
```

File: payment-instruments/card/features/optional-features/index.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+---
+title: Optional features
+redirect_from:
+estimated_read: -
+description: |
+  Welcome to Optional Features - a subsection of Features.
+  This section has extented code examples and features that were not
+  covered by the other subsections.
```
Comment (asbjornu) [inline payment-instruments/card/features/optional-features/index.md:8]: I'm not happy about my suggested description, but I think we can be a bit terser and on point. We should also have a card here for each optional feature, no?

```suggestion
  This section details the features of Card that are optional.
```

File: payment-instruments/card/features/optional-features/payout.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
+---
+title: Payout
+redirect_from:
+estimated_read: 3
+description: |
+  Welcome to Payout.
```
Comment (asbjornu) [inline payment-instruments/card/features/optional-features/payout.md:6]: Do we need to say "Welcome to …" everywhere?

File: payment-instruments/card/features/optional-features/payout.md [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+  This section has extented code examples and features that were not
+  covered by the other subsections.
```
Comment (asbjornu) [inline payment-instruments/card/features/optional-features/payout.md:8]: Repeating this in every description isn't useful. Please describe the feature, here being "Payout", instead.

File: payment-instruments/card/features/technical-reference/index.md [hunk 8]
```diff
@@ -0,0 +1,10 @@
+---
+title: Technical Reference
+redirect_from:
+estimated_read: -
+description: |
+  Welcome to Technical Reference - a subsection of Features.
+  This section has extented code examples and features that were not
+  covered by the other subsections.
```
Comment (asbjornu) [inline payment-instruments/card/features/technical-reference/index.md:8]:
```suggestion
  This section is a technical reference for the requests and responses
  you will encounter when integrating against the Card Payment Instrument.
```

**Review by asbjornu — CHANGES_REQUESTED** [review]
This is massive work on an epic scale and very well executed for the most part. Really good job! 👏🏼 🎉 There are only a few general themes I would like addressed:

1. The descriptions mostly don't make much sense. Please improve upon them so they describe each page more clearly.
2. `core-features` should be renamed to just `core`.
3. All `index.md` pages needs to have a `redirect_from` directive to redirect from the old `other-features` page.
4. All `index.md` pages needs to list the sub-pages as cards, or else they seem very empty.
5. Many links aren't working. Please fix.

File: features/index.md (hunk 3 above)
Comment (arebra) [reply features/index.md:1]: I am actually not sure why Marina created this folder. This features folder doesn't seem to be used for anything, as the links are outdated and we have features folders in the checkout, payment menu and each payment instrument sections.

Comment (asbjornu) [reply features/index.md:1]: Then delete it?

**Review by asbjornu — APPROVED** [review]
🎉 👏🏼

**Pull request merged**